examples/chicane/run_chicane.py

```diff
@@ -48,17 +48,18 @@
 
 # design the accelerator lattice
 ns = 25  # number of slices per ds in the element
-rc = 10.35  # bend radius (meters)
+rc = 10.3462283686195526  # bend radius (meters)
```

Comment:
We could use Python math here to make it a bit clearer how those (projected) parameters come to play.